client/src/plus/deep-dives/byline/index.tsx

```diff
@@ -26,7 +26,7 @@ export function Byline({
       <div className="author-byline">
         <h3 className="author-name">{author}</h3>
         <p className="author-description">
-          {authorDescription}
+          {authorDescription}{" "}
```

Comment:
I noticed there was a hack in the *use* of `<Byline>` that forced in a whitespace character. That's the wrong way to do it because it's only a matter of time till it doesn't work. Some day in some future sure the strings, in the caller of the component, won't be hardcoded but come from a database.